src/main/markdown/release-notes.md

```diff
@@ -59,10 +59,79 @@ The GWT Release Notes
 
 <a id="Release_Notes_Current"></a>
 
-<!-- ## <a id="Release_Notes_2_8_0_RC1"></a> Release Notes for 2.8.0 (RC1)
+## <a id="Release_Notes_2_8_0_RC1"></a> Release Notes for 2.8.0 (RC1)
+
+### Highlights
+- Partial support for Java 8 standard library APIs (see below for full list).
+- Fix memory leak with Java 8 compilation.
+- Source level set to Java 8.
+- Static and default methods in interfaces aren't visible to generators. If you want to take advantage of those Java-8isms, you're encouraged to switch to an annotation processor. This could break existing build if an interface is changed to turn a non-default method into a default method.
 
 ### Deprecations
-- Classic dev mode deprecated.You can switch back to dev mode for your test cases temporarily by passing gwt.args='-devMode'. -->
+- Classic dev mode deprecated.You can switch back to dev mode for your test cases temporarily by passing gwt.args='-devMode'.
+- Deprecate JSR 303 Bean Validation support (javax.validation, c.g.g.validation). The code hasn't been maintained for years.
+
+### Breaking changes
+- GIN needs a workaround now: gwtproject/gwt#9311
```

Comment:
Make a real link to the bug (same below), or will it somehow be auto converted to a link?